docs/src/integrations/exchange.md

```diff
@@ -695,7 +708,7 @@ lamports).
 
 Template `spl-token transfer` command for a withdrawal:
```

Comment:
Also add some best practices like

- check owner address having more than 0 lamports
- check given withdrawal address isn't token holding account, and actually system account.